tsconfig.json

```diff
@@ -1,8 +1,13 @@
 {
   "extends": "@tsconfig/node18/tsconfig.json",
   "exclude": ["packages"],
+  "include": ["src/**/*", "**/*.ts", "**/*.tsx"],
```

Comment:
is this really desired? you are adding all the files in `src` but then all the files in the root and under that are `tsx?` ... maybe change to `["src/*"]`

Reply:
I see how this is redundant.  I changed it to ["src/*"]

Reply:
Upon closer examination of the file directory the correct path for include is packages/web/src/**/*  Exlude was also being overly aggressive in the files it was filtering out so I removed exclude.